backend/routers/plugins.py

```
@@ -109,3 +116,129 @@ def get_plugin_money_made(plugin_id: str):
+@router.post('/v1/plugins/add', tags=['v1'])
+def add_plugin(plugin_data: str = Form(...), file: UploadFile = File(...), uid=Depends(auth.get_current_user_uid)):
+    data = json.loads(plugin_data)
+    data['approved'] = False
+    data['id'] = data['name'].replace(' ', '-').lower()
+    data['uid'] = uid
+    if data['private']:
+        data['id'] = data['id'] + '-private'
+    else:
+        if plugin_id_exists_db(data['id']):
+            data['id'] = data['id'] + '-' + ''.join([str(random.randint(0, 9)) for _ in range(5)])
+    os.makedirs(f'_temp/plugins', exist_ok=True)
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)